### ISSUE TITLE
brl-disable error when specifiying non-existent stratum

### ISSUE THREAD

**Issue by person4268** [bedrocklinux/bedrocklinux-userland#152]
When I run brl disable with a stratum that doesn't exist, I get this output:
```
/bedrock/libexec/brl-disable: line 45: stratum: parameter not set
ERROR: Unexpected error occurred.
```
I've fixed this in #150, but it hasn't been merged.

**Comment by paradigm** [comment]
Thanks!  I'll close this once your fix is released.

**Comment by paradigm** [comment]
Should be fixed in the just released 0.7.11.  Thanks again!
